components/brave_new_tab_ui/components/default/binance/index.tsx

```diff
@@ -605,10 +606,10 @@ class Binance extends React.PureComponent<Props, State> {
   }
 
   renderTitleTab () {
-    const { onShowContent } = this.props
+    const { onShowContent, stackPosition } = this.props
 
     return (
-      <StyledTitleTab onClick={onShowContent}>
+      <StyledTitleTab onClick={onShowContent} stackPosition={stackPosition}>
```

Comment:
nit (continued): and here do `<StyledTitleTab ...getWidgetTitleProps(this.props)>`

_...but that's if you didn't want to do the repetition ;-)_